frontends/web/src/routes/bitboxbase/bitboxbase.tsx

```diff
@@ -238,11 +238,7 @@ class BitBoxBase extends Component<Props, State> {
 
         // Only create a new websocket if the bitboxBaseID changed.
         if (this.props.bitboxBaseID !== this.state.bitboxBaseID && this.props.bitboxBaseID) {
-<<<<<<< HEAD
             this.setState({ bitboxBaseID : this.props.bitboxBaseID });
```

Comment:
Note: one rule removed a space before the colon and the other added a space before the closing bracket, change
`this.setState({ bitboxBaseID : this.props.bitboxBaseID });` to
`this.setState({ bitboxBaseID: this.props.bitboxBaseID });`

Reply:
rebased